examples/ruby/hellowithdata.rb

```diff
@@ -0,0 +1,5 @@
+def handler(request)
+  payload = request.body.read
+  puts JSON.parse(payload)
+  return payload
+end
```

Comment:
I'd add a `rescue` class for `JSON::ParserError` exceptions

Reply:
good idea but I want to keep it simple like the other examples